rio_viz/raster.py

```diff
@@ -46,6 +48,24 @@ def _get_name(ix):
     return bounds, center, minzoom, maxzoom, band_descriptions, data_type, cmap
 
 
+# From https://gist.github.com/geospatial-jeff/baf32f44bf2fbd073498e3d92f82523b#file-rio_tiler_asyncio-py-L21
+async def run_in_threadpool(func: Callable, *args: Any, **kwargs: Any):
```

Comment:
I think it would be better to use `from starlette.concurrency import run_in_threadpool` instead.